File: sentimental-mario-less/mario.py
from cs50 import get_int

# Getting input, height, from the user
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# Printing desired pattern
cpy = height
for i in range(height):
    j = 1
    while j < cpy:
        print(" ", end="")
        j += 1
    while j >= cpy and j <= height:
        print("#", end="")
        j += 1
    cpy -= 1
    print("")

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

// Declaring functions
int input(void);
void print_pattern(int n);

// Main Function
int main(void)
{
    int n = input();
    print_pattern(n);
}

// Taking Input
int input(void)
{
    int width;
    do
    {
        width = get_int("Width: ");
    }
    while (width <= 0 || width > 8);
    return width;
}

// Printing Pattern
void print_pattern(int n)
{
    for (int i = 0; i < n; i++)
    {
        for (int j = i; j < (n - 1); j++)
        {
            printf(" ");
        }
        for (int j = n; j >= (n - i); j--)
        {
            printf("#");

        }
        printf("\n");
    }
}